### PR TITLE
Remove Microgateway Labels

### PR DESCRIPTION
**Pull request by renuka-fernando**
#### Description
- Remove Rest APIs for MG labels
- Remove UIs of MG labels (Dev Portal)

TODO: GitHub action should be reverted back to WSO2 master
- Related Integration Test changes: https://github.com/wso2/product-apim/pull/10254

#### Related Issues
- https://github.com/wso2/product-apim/issues/10253

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,12 +1,6 @@
 name: API Manager Build with Test
 
-on:
```
Comment (tharindu1st) [inline .github/workflows/main.yml:3]: revert the change in main.yaml

**Review by tharindu1st — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (renuka-fernando) [reply .github/workflows/main.yml]: reverted with commit: https://github.com/wso2/carbon-apimgt/pull/10010/commits/83725d07cc8d603f6e94fa10d13f1c421eda7b02

**Review by tmkasun — APPROVED** [review]

**Review by 1akshitha — APPROVED** [review]

**Comment by renuka-fernando** [comment]
Failed test cases in GitHub Actions
```log
APIProductCreationTestCase.testCreateAndInvokeApiProduct
──────────────────────────────────────────────────────────────────────────────────────────
 expected [200] but found [401] ──────────────────────────────────────────────────────────────────────────────────────────
java.lang.AssertionError: expected [200] but found [401] at org.wso2.am.integration.tests.apiproduct.APIProductCreationTestCase.testCreateAndInvokeApiProduct(APIProductCreationTestCase.java:152)


AccessibilityOfOldAPIAndCopyAPIWithReSubscriptionTestCase.testInvokeNewAPIAfterSubscribeTheNewVersion ──────────────────────────────────────────────────────────────────────────────────────────
Response code mismatched when invoke new api after subscribe the new version expected [200] but found [401] ──────────────────────────────────────────────────────────────────────────────────────────
java.lang.AssertionError: Response code mismatched when invoke new api after subscribe the new version expected [200] but found [401] at org.wso2.am.integration.tests.api.lifecycle.AccessibilityOfOldAPIAndCopyAPIWithReSubscriptionTestCase.testInvokeNewAPIAfterSubscribeTheNewVersion(AccessibilityOfOldAPIAndCopyAPIWithReSubscriptionTestCase.java:217)
```

Test cases ran in local environment:
```xml
<test name="apim-integration-tests-api-product" preserve-order="true" parallel="false" group-by-instances="true">
    <classes>
        <class name="org.wso2.am.integration.tests.api.revision.APIProductRevisionTestCase"/>
        <class name="org.wso2.am.integration.tests.apiproduct.APIProductCreationTestCase"/>
    </classes>
</test>

<test name="apim-integration-tests-api-lifecycle" preserve-order="true" parallel="false" group-by-instances="true">
    <classes>
        <class name="org.wso2.am.integration.tests.api.lifecycle.AccessibilityOfDeprecatedOldAPIAndPublishedCopyAPITestCase"/>
        <class name="org.wso2.am.integration.tests.api.lifecycle.AccessibilityOfOldAPIAndCopyAPIWithOutReSubscriptionTestCase"/>
        <class name="org.wso2.am.integration.tests.api.lifecycle.AccessibilityOfOldAPIAndCopyAPIWithReSubscriptionTestCase"/>
        <class name="org.wso2.am.integration.tests.api.lifecycle.AccessibilityOfRetireAPITestCase"/>
    </classes>
</test>
```
#### Build Succeeded in local setup
```log
[INFO] Tests run: 40, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1,505.222 s - in TestSuite
[INFO]
[INFO] Results:
[INFO]
[INFO] Tests run: 40, Failures: 0, Errors: 0, Skipped: 0
[INFO]
[INFO]
[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ org.wso2.carbon.am.integration.backend.test ---
[INFO] Building jar: /Users/renuka/git/product-apim/modules/integration/tests-integration/tests-backend/target/org.wso2.carbon.am.integration.backend.test-4.0.0-SNAPSHOT.jar
[INFO]
[INFO] --- maven-jar-plugin:2.4:test-jar (default) @ org.wso2.carbon.am.integration.backend.test ---
[INFO] Building jar: /Users/renuka/git/product-apim/modules/integration/tests-integration/tests-backend/target/org.wso2.carbon.am.integration.backend.test-4.0.0-SNAPSHOT-tests.jar
[INFO]
[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ org.wso2.carbon.am.integration.backend.test ---
[INFO] Installing /Users/renuka/git/product-apim/modules/integration/tests-integration/tests-backend/target/org.wso2.carbon.am.integration.backend.test-4.0.0-SNAPSHOT.jar to /Users/renuka/.m2/repository/org/wso2/am/org.wso2.carbon.am.integration.backend.test/4.0.0-SNAPSHOT/org.wso2.carbon.am.integration.backend.test-4.0.0-SNAPSHOT.jar
[INFO] Installing /Users/renuka/git/product-apim/modules/integration/tests-integration/tests-backend/pom.xml to /Users/renuka/.m2/repository/org/wso2/am/org.wso2.carbon.am.integration.backend.test/4.0.0-SNAPSHOT/org.wso2.carbon.am.integration.backend.test-4.0.0-SNAPSHOT.pom
[INFO] Installing /Users/renuka/git/product-apim/modules/integration/tests-integration/tests-backend/target/org.wso2.carbon.am.integration.backend.test-4.0.0-SNAPSHOT-tests.jar to /Users/renuka/.m2/repository/org/wso2/am/org.wso2.carbon.am.integration.backend.test/4.0.0-SNAPSHOT/org.wso2.carbon.am.integration.backend.test-4.0.0-SNAPSHOT-tests.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 25:50 min
[INFO] Finished at: 2021-03-18T12:44:30+05:30
[INFO] ------------------------------------------------------------------------
```

Full Log: [9.0.66-SNAPSHOT-remove-mg-labels.tar.zip](https://github.com/wso2/carbon-apimgt/files/6162098/9.0.66-SNAPSHOT-remove-mg-labels.tar.zip)

**Review by tharindu1st — APPROVED** [review]

**Review by malinthaprasan — APPROVED** [review]
